modules/core/src/models/storage.ts

```diff
@@ -36,11 +36,35 @@ export interface StorageApplicationSettingsItemsInterface {
     recentApplications: string[];
 }
 
+/**
+ * Interface for the dev portal preferences.
+ */
+export interface StorageDevPortalPreferencesInterface {
+    /**
+     * Help panel preferences.
+     */
+    helpPanel: StorageHelpPanelPreferencesItemsInterface;
+}
+
+
+/**
+ * Interface for the help panel preferences.
+ */
+export interface StorageHelpPanelPreferencesItemsInterface {
+    /**
+     * Is help panel pinned.
+     */
+    isPinned: boolean;
+}
+
 /**
  * Interface for the user preferences object.
+ * TODO: Revisit the structure. Modify this interface to accept a generic portal settings type and
+ *  assign it to a dynamic key. Ex: [ portal: string ]: T;
  */
 export interface StorageIdentityUserPreferencesInterface {
     applicationPreferences: StorageApplicationSettingsItemsInterface;
+    devPortal: StorageDevPortalPreferencesInterface;
```

Comment:
Shall we avoid these naming usage? Not to mention devPortal or any. Just generic terms

Reply:
Yes this structure needs to be refactored. Lets get this fixed. I have created an issue[1] to track the progress of the task.

[1] https://github.com/wso2/identity-apps/issues/776